### PR TITLE
Prevent issue with bootstrapping ns from a clean repl for notebooks

### PR DESCRIPTION
**Pull request by djblue**
## What has changed?

See #2498, fixes issue caused by #2409

## My Calva PR Checklist

I have:

- [x] Read [How to Contribute](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Contribute#before-sending-pull-requests).
- [x] Directed this pull request at the `dev` branch. (Or have specific reasons to target some other branch.)
- [x] Made sure I have changed the PR base branch, so that it is not `published`. (Sorry for the nagging.)
- [x] Made sure there is an issue registered with a clear problem statement that this PR addresses, (created the issue if it was not present).
    - [x] Updated the `[Unreleased]` entry in `CHANGELOG.md`, linking the issue(s) that the PR is addressing.
- [ ] Figured if **anything** about the fix warrants tests on Mac/Linux/Windows/Remote/Whatever, and either tested it there if so, or mentioned it in the PR.
- [ ] Added to or updated docs in this branch, if appropriate
- [x] Tests
  - [x] Tested the particular change (starting from a clean/empty repl)
  - [ ] Figured if the change might have some side effects and tested those as well.
- [ ] Formatted all JavaScript and TypeScript code that was changed. (use the [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or run `npm run prettier-format`)
- [x] Confirmed that there are no linter warnings or errors (use the [eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), run `npm run eslint` before creating your PR, or run `npm run eslint-watch` to eslint as you go).

Ping @pez, @bpringe, @corasaurus-hex, @Cyrik

### CODE REVIEW

**Comment by PEZ** [comment]
Thanks! I've fixed the issue with docs not failing. You can merge `dev` and things should pass. Let us know when this is ready for merging.

**Comment by djblue** [comment]
@PEZ, I updated this branch with dev and retested Calva Notebooks starting from a clean REPL to ensure this fixes the issue. This PR should be ready to go 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Changes to Calva.
 ## [Unreleased]
 
 - [Make the debugger present structural variables as traversable structures](https://github.com/BetterThanTomorrow/calva/issues/2494)
+- [Fix regression caused by #2409 for notebooks with uninitialized namespaces](https://github.com/BetterThanTomorrow/calva/issues/2498)
```
Comment (PEZ) [inline CHANGELOG.md]:
```suggestion
- Fix: [Regression caused by #2409 for notebooks with uninitialized namespaces](https://github.com/BetterThanTomorrow/calva/issues/2498)
```

**Comment by PEZ** [comment]
Thanks! ❤️

**Pull request merged**